TiDbBackupScheduler databaseName not included in BR when creating the backup (Azure)

## Bug Report

**What version of Kubernetes are you using?**
1.22.11

**What version of TiDB Operator are you using?**
TiDB Operator Version: version.Info{GitVersion:"v1.3.7", GitCommit:"e919bd572726efff550411f2623fb1d408742f46", GitTreeState:"clean", BuildDate:"2022-08-01T06:18:04Z", GoVersion:"go1.13.8", Compiler:"gc", Platform:"linux/a
md64"}


**What did you do?**
Create a TiDbBackupScheduler

```
apiVersion: pingcap.com/v1alpha1
kind: BackupSchedule
metadata:
  name: tidb-backup-schedule
  namespace: tidb
spec:
  backupTemplate:
    azblob:
      accessTier: Cool
      container: tidb
      secretName: storage-account-backup
    backupMode: snapshot
    br:
      cluster: tidb
      clusterNamespace: tidb
    cleanPolicy: Delete
    resources: {}
  maxBackups: 10
  schedule: 0 */1 * * *
```

**What did you expect to see?**
The BR uses the backupName as a folder

**What did you see instead?**
```
Create rclone.conf file.
/tidb-backup-manager backup --namespace=tidb **--backupName=tidb-backup-schedule-2022-10-04t15-00-00** --tikvVersion=v5.4.0
I1004 15:00:26.672544       9 backup.go:72] start to process backup tidb/tidb-backup-schedule-2022-10-04t15-00-00
I1004 15:00:26.694151       9 backup_status_updater.go:86] Backup: [tidb/tidb-backup-schedule-2022-10-04t15-00-00] updated successfully
E1004 15:00:26.700888       9 backup_status_updater.go:89] Failed to update backup [tidb/tidb-backup-schedule-2022-10-04t15-00-00], error: Operation cannot be fulfilled on backups.pingcap.com "tidb-backup-schedule-2022-10-04t15-00-00": the object has been modified; please apply your changes to the latest version and try again
I1004 15:00:26.731027       9 backup_status_updater.go:86] Backup: [tidb/tidb-backup-schedule-2022-10-04t15-00-00] updated successfully
I1004 15:00:26.731073       9 backup.go:69] Running br command with args: [backup full --pd=tidb-pd.tidb:2379 --**storage=azure://tidb/** --azblob.access-tier=Cool]
****** Other log ****
I1004 15:00:26.932980       9 backup.go:93] [2022/10/04 15:00:26.932 +00:00] [ERROR] [backup.go:40] ["failed to backup"] [error="backup meta file exists in azure://tidb//backupmeta, there may be some backup files in the path already, please specify a correct backup directory!: [BR:Common:ErrInvalidArgument]invalid argument"] [errorVerbose="[BR:Common:ErrInvalidArgument]invalid argument\nbackup meta file exists in azure://tidb//backupmeta, there may be some backup files in the path already, please specify a correct backup directory!\ngithub.com/pingcap/tidb/br/pkg/backup.(*Client).SetStorage\n\t/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/br/br/pkg/backup/client.go:179\ngithub.com/pingcap/tidb/br/pkg/task.RunBackup\n\t/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/br/br/pkg/task/backup.go:261\nmain.runBackupCommand\n\t/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/br/br/cmd/br/backup.go:39\nmain.newFullBackupCommand.func1\n\t/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/br/br/cmd/br/backup.go:108\ngithub.com/spf13/cobra.(*Command).execute\n\t/nfs/cache/mod/github.com/spf13/cobra@v1.0.0/command.go:842\ngithub.com/spf13/cobra.(*Command).ExecuteC\n\t/nfs/cache/mod/github.com/spf13/cobra@v1.0.0/command.go:950\ngithub.com/spf13/cobra.(*Command).Execute\n\t/nfs/cache/mod/github.com/spf13/cobra@v1.0.0/command.go:887\nmain.main\n\t/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/br/br/cmd/br/main.go:56\nruntime.main\n\t/usr/local/go/src/runtime/proc.go:225\nruntime.goexit\n\t/usr/local/go/src/runtime/asm_amd64.s:1371"] [stack="main.runBackupCommand\n\t/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/br/br/cmd/br/backup.go:40\nmain.newFullBackupCommand.func1\n\t/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/br/br/cmd/br/backup.go:108\ngithub.com/spf13/cobra.(*Command).execute\n\t/nfs/cache/mod/github.com/spf13/cobra@v1.0.0/command.go:842\ngithub.com/spf13/cobra.(*Command).ExecuteC\n\t/nfs/cache/mod/github.com/spf13/cobra@v1.0.0/command.go:950\ngithub.com/spf13/cobra.(*Command).Execute\n\t/nfs/cache/mod/github.com/spf13/cobra@v1.0.0/command.go:887\nmain.main\n\t/home/jenkins/agent/workspace/build-common/go/src/github.com/pingcap/br/br/cmd/br/main.go:56\nruntime.main\n\t/usr/local/go/src/runtime/proc.go:225"]
```


As you can see, the BR  parameters don't include the backupName as folder. So at the end it fails because it try to create the backup in the root container (where a previous backup has already created their files)

--- a/pkg/backup/backupschedule/backup_schedule_manager.go
+++ b/pkg/backup/backupschedule/backup_schedule_manager.go
@@ -221,6 +221,8 @@ func buildBackup(bs *v1alpha1.BackupSchedule, timestamp time.Time) *v1alpha1.Bac
 			backupSpec.S3.Prefix = path.Join(backupSpec.S3.Prefix, backupPrefix)
 		} else if backupSpec.Gcs != nil {
 			backupSpec.Gcs.Prefix = path.Join(backupSpec.Gcs.Prefix, backupPrefix)
+		} else if backupSpec.Azblob != nil {
+			backupSpec.Azblob.Prefix = path.Join(backupSpec.Azblob.Prefix, backupPrefix)
 		} else if backupSpec.Local != nil {
 			backupSpec.Local.Prefix = path.Join(backupSpec.Local.Prefix, backupPrefix)
 		}

